frontend/src/components/maps/leaflet/MapProperties.test.tsx

```diff
@@ -3,85 +3,129 @@ import renderer from 'react-test-renderer';
 import { ParcelPopupView } from 'components/maps/ParcelPopupView';
 import { IParcelDetail, IParcel } from 'actions/parcelsActions';
 import Map from './Map';
-import { Marker } from 'react-leaflet'
-import {mount, shallow} from 'enzyme'
+import { Marker } from 'react-leaflet';
+import { mount, shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import Enzyme from 'enzyme'
+import Enzyme from 'enzyme';
 import { render } from '@testing-library/react';
 
 Enzyme.configure({ adapter: new Adapter() });
- 
+
 // This will spoof the active parcel (the one that will populate the popup details)
-const mockDetails = {
-    id: 1,
-    pid: '000-000-000',
-    latitude: 48,
-    longitude: 123,
-    propertyStatus: 'active',
-    propertyClassification: 'Core Operational',
-    description: 'test',
-    assessedValue: 1000000,
-    address: {
-        line1: '1234 mock Street',
-        line2: 'N/A',
-        city: 'Victoria',
-        province: 'BC',
-        postal: 'V1V1V1',
-    },
-    landArea: 'unknown',
-    landLegalDescription: 'test',
-    buildings: [],
-    agency: 'FIN',
-} as IParcelDetail
+const mockDetails: IParcelDetail = {
+  id: 1,
+  pid: '000-000-000',
+  latitude: 48,
+  longitude: 123,
+  propertyStatus: 'active',
+  classification: 'Core Operational',
+  description: 'test',
+  assessedValue: 1000000,
+  address: {
+    line1: '1234 mock Street',
+    line2: 'N/A',
+    city: 'Victoria',
+    province: 'BC',
+    postal: 'V1V1V1',
+  },
+  landArea: 'unknown',
+  landLegalDescription: 'test',
+  buildings: [],
+  agency: 'FIN',
+};
```

Comment:
Review: This object was using old properties no longer present in **IParcelDetail**